src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -438,8 +438,8 @@ export default {
         // is this a Future Effective Incorp App?
         const isIaFutureEffective = this.isFutureEffective(filing)
 
-        // if it's not a FE IA, is it paid?
-        const isPaid = !isIaFutureEffective && (header.status === FilingStatus.PAID)
+        // is this a paid filing?
+        const isPaid = (header.status === FilingStatus.PAID)
```

Comment:
There was no reason to have isPaid mutually exclusive with isIaFutureEffective. Really, in some cases, they can both be true. The only caveat is that, depending on what needs to display, the template should first check if isIaFutureEffective, else check if isPaid... the template already does that so we're good!